meta/f710ac3c.csv

```diff
@@ -1069,3 +1069,4 @@ Id,FileID,Domain,RepoName,FilePath,LineStart:LineEnd,GroundTruth,WithWords,Value
 132526,cdecae61,GitHub,f710ac3c,data/f710ac3c/test/ck.txt,76:76,T,F,,,F,F,,,,,0.00,,F,F,F,Other
 132527,cdecae61,GitHub,f710ac3c,data/f710ac3c/test/ck.txt,78:78,T,F,,,F,F,,,,,0.00,,F,F,F,Other
 132126,eb20de75,GitHub,f710ac3c,data/f710ac3c/test/bh.py,922:922,T,F,,,F,F,,,,,0.00,,F,F,F,Other
+132673,d67e41a7,GitHub,f710ac3c,data/f710ac3c/test/cj.py,163:163,T,F,,,F,F,,,,,0.00,,F,F,F,Password
```

Comment:
As I know our `MIN_VALUE_LENGTH` is 4, but it seems it's 3.
Does it detected by CredSweeper?

Reply:
``\`` is a charachter too

Reply:
Okay.. I see.